Update variable fonts article

RoelN opened the pull request:
This is an update to [Introduction to variable fonts on the web](https://web.dev/variable-fonts/) by @davelab6 and @roeln. Dave and I paired on this, and I added some animations afterwards, and now submitting this PR.

This PR needs one more revision by @davelab6 to address the last TODOs.

Changes proposed in this pull request:

- General updates and rewrites
- Updated `@font-face` code examples
- Updated examples to modern GF fonts
- Updated fallbacks and browser support
- Add CSS custom properties workaround for `font-variation-settings`
- Images and movies have been optimised

## TODO:
- [ ] TODOs (see "TODO" and "RN" comments in document)
  - [ ] Add Dave Crossland and Roel Nieskens to authors
  - [ ] @davelab6 TODO: [Print industry reference](https://github.com/GoogleChrome/web.dev/blame/7ac29dbea9e4e7bd47faac2f775cbf272a87595f/src/site/content/en/blog/variable-fonts/index.md#L53)
  - [ ] @davelab6 TODO: [Simplify language of intruction](https://github.com/GoogleChrome/web.dev/blame/7ac29dbea9e4e7bd47faac2f775cbf272a87595f/src/site/content/en/blog/variable-fonts/index.md#L95)
  - [ ] @davelab6 TODO: ["Restart here"](https://github.com/GoogleChrome/web.dev/blame/7ac29dbea9e4e7bd47faac2f775cbf272a87595f/src/site/content/en/blog/variable-fonts/index.md#L139)
  - [ ] @davelab6 TODO: [explain opsz, and px/pt snakepit](https://github.com/GoogleChrome/web.dev/blame/7ac29dbea9e4e7bd47faac2f775cbf272a87595f/src/site/content/en/blog/variable-fonts/index.md#L139)
- [x] Add animation for Amstelvar "Finesse" example
  - [ ] Check text explaining Amstelvar "Finesse" example, not sure my placeholder covers everything
- [ ] Move image/movie media to https://storage.googleapis.com/web-dev-assets/variable-fonts

RoelN commented:
@googlebot I signed it!

davelab6 commented:
@dberlow how would you like to handle the finesse example?

kaycebasques commented:
Hey @davelab6 & @RoelN I just want to do an initial check that @tomayac & @mustafa-x have agreed to hand off this article to you?

dberlow commented:
github.com/TypeNetwork/Amstelvar/blob/master/docs/xtra%20animation%20demo.pdf

On Tue, Jul 7, 2020 at 8:02 AM Dave Crossland <notifications@github.com>
wrote:

> @dberlow <https://github.com/dberlow> how would you like to handle the
> finesse example?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleChrome/web.dev/pull/3445#issuecomment-654807608>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAO5VDRE75PSVVMMSAZQ753R2MFDPANCNFSM4OSS657A>
> .
>

tomayac commented:
> Hey @davelab6 & @RoelN I just want to do an initial check that @tomayac & @mustafa-x have agreed to hand off this article to you?

I consider @mustafa-x the content owner, I helped with the technical migration to web.dev. Mustafa, your call.

davelab6 commented:
@kaycebasques yes, @mustafa-x agreed to hand the page over (back in Q1). Since @RoelN and @dberlow are both commissioned by me (Google Fonts) to do this work, the copyrights are already assigned to Google LLC. 

@dberlow thanks, that PDF looks good to me. @RoelN is on summer vacations right now, I expect he can implement this when he gets back. 

@tomayac would you be able to help with (confirming) this TODO item that Roel mentioned at the top? :)

> Move image/movie media to storage.googleapis.com/web-dev-assets/variable-fonts, if the build step doesn't do this automatically

tomayac commented:
> @tomayac would you be able to help with (confirming) this TODO item that Roel mentioned at the top? :)
> 
> > Move image/movie media to storage.googleapis.com/web-dev-assets/variable-fonts, if the build step doesn't do this automatically

According to the [web.dev handbook](https://web.dev/handbook/markup-media/#video-hosted-on-web.dev:~:text=Once%20you%20have%20a%20video%20ready%2C,the%20web.dev%20Google%20Cloud%20Storage%20bucket.), the content reviewer (i.e., @kaycebasques) will help you move the assets.

mustafa-x commented:
im cool with handing over the article :)

kaycebasques commented:
Thanks for the confirmations, everybody. I'll help get these updates over the finish line.

RoelN commented:
Thanks for your patience, all! I'm [working on that last animation](https://github.com/TypeNetwork/Amstelvar/pull/182) and once approved by @dberlow or @davelab6 I'll add them to this document.

dberlow commented:
Okay with me, thanks!

On Aug 4, 2020, at 10:49 AM, Roel Nieskens <notifications@github.com> wrote:

﻿
Thanks for your patience, all! I'm working on that last animation and once approved by @dberlow or @davelab6 I'll add them to this document.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.

RoelN commented:
Animations have been added. I updated the [TODO list](https://github.com/GoogleChrome/web.dev/pull/3445#issue-445265403) for this PR. Some are just suggestions from me, if we don't want to follow them we can just check off these items.

Let me know if anything else is required of me :-)

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -8,360 +8,710 @@ authors:
-and [Fallbacks](#fallbacks).
+In this article, we will look at what variable fonts are, the benefits they
+offer, and how we can use them in our work. First, let's review how typography
+works on the web, and what innovations variable fonts bring.
```
Please put the browser compatibility data back in. We know from the [MDN DNA survey](https://mdn-web-dna.s3-us-west-2.amazonaws.com/MDN-Web-DNA-Report-2019.pdf) (page 19) that browser compatibility is a top concern for web developers. We put it at the top of articles to convince developers that the rest of the content is worth their time (it's very frustrating to read a guide only to find out at the end that an API is only supported in Chrome).

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -8,360 +8,710 @@ authors:
-    Christian Robertson.
+  <img src="roboto-family-specimen.png" style="max-width: 100%;" alt="A specimen composition and list of different styles of the Roboto family">
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Above: Left, a specimen of the Roboto typeface family. Right, named styles within the family.
```
```suggestion
    Left: a specimen of the Roboto typeface family. Right: named styles within the family.
```
I don't think `Above` is necessary because it's pretty clearly styled as the caption for the image

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -8,360 +8,710 @@ authors:
   </figcaption>
 </figure>
 
+## Challenges for the Designer and Developer
```
```suggestion
## Challenges for the designer and developer
```
https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -8,360 +8,710 @@ authors:
+Unlike books or magazines, which can have as many fonts on a page as they like, a website will have to download every single font it uses. This has been a sticking point for richer typographic experiences, as each font comes at a cost.
+
+With traditional web fonts... etc.
+-->
```
Have you resolved these comments? Please delete if so.

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -8,360 +8,710 @@ authors:
+-->
+
+When a designer creates a print project they face some constraints, such as the
+physical size of the page layout, the number of colors they can use is
+determined by the kind of printing press that will be used, and so on. But they
```
```suggestion
physical size of the page layout, the number of colors they can use (which is
determined by the kind of printing press that will be used), and so on. But they
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -8,360 +8,710 @@ authors:
+cost. With traditional web fonts, each style used in our designs requires users
+to download a separate font file, which increases latency and page rendering
+time. Only including the Regular and Bold styles, plus their italic
+counterparts, can amount to 500k or more of font data. This is even before we
```
```suggestion
counterparts, can amount to 500 KB or more of font data. This is even before we
```
I can't find a reference in style guide but I'm pretty sure we use KB for kilobytes and Kb for kilobits

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -8,360 +8,710 @@ authors:
+to download a separate font file, which increases latency and page rendering
+time. Only including the Regular and Bold styles, plus their italic
+counterparts, can amount to 500k or more of font data. This is even before we
+have dealt how the fonts are rendered, and the fallback or delayed-loading
```
```suggestion
have dealt with how the fonts are rendered, and the fallback or delayed-loading
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -8,360 +8,710 @@ authors:
+time. Only including the Regular and Bold styles, plus their italic
+counterparts, can amount to 500k or more of font data. This is even before we
+have dealt how the fonts are rendered, and the fallback or delayed-loading
+patterns we are going to use (such as ["FOIT" and
+"FOUT."](https://www.zachleat.com/web/fout-vs-foit/))
```
```suggestion
patterns we are going to use (such as [FOIT and
FOUT](https://www.zachleat.com/web/fout-vs-foit/)).
```
FOIT and FOUT aren't delayed-loading patterns, are they? They're more like undesirable unintended side effects? `delayed-loading patterns` sounds like something we do intentionally.

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -8,360 +8,710 @@ authors:
+sentence below makes it sound the previous sentence lists "few capabilities"?
+Maybe something like: "Since you'd have to load a new font file for every style
+(or combinations of styles), many web developers choose not to use these
+capabilities, reducing the reading experience of their users." --> When using so
```
Are you going to resolve this comment?

RoelN commented:
@kaycebasques Thanks for the review! I addressed all of the issues, except adding us as authors. Will do so tomorrow.

I removed the TODOs for @davelab6 so the document is clean now, but Dave said he wants to take a look anyway this week. So he'll check in later with either edits or LGTMs!

kaycebasques commented:
SGTM, I'll wait for @davelab6's LGTM before reviewing any further

RoelN commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -8,360 +8,710 @@ authors:
   - mustafakurtuldu
   - thomassteiner
```
Added @davelab6 and myself in e5a4bff2235222a401d2cd374eaa2ea04c5393bd :-)

RoelN commented:
Added myself as author, took the liberty to add @davelab6 as well.

As for those "Alex Recommends" comments, they can be ignored. ("...just as...", "hero image")

kaycebasques commented:
@davelab6 waiting on your LGTM

@RoelN you've got a merge conflict in the authors file

kaycebasques commented:
Talked with @davelab6, he's not going to have time to review. I'll take it from here.

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
+can connect the default style through to a Bold master style. Any individual
+style can be located along an axis, and is called an 'instance' of the variable
+font. Some instances are named by the font developer, for example Weight axis
+location 600 is called SemiBold.
```
@RoelN is there any other phrase we can use other than `master` here? My org is asking me to avoid that word because of it's political/historical connotations (master/slave)

RoelN replied:
Done!

dberlow commented:
My opinion is you can remove the word in all cases. 

In the case of using “Default Master”, it can simply be removed, leaving “Default‘ or ‘‘Default Regular”. In cases where it’s used to describe part of an axis, “bold master” can change to “bold style”, and “masters” in all other references can be replaced by “style” or “styles”.

Hope that’s helpful.

On Sep 2, 2020, at 6:37 PM, Kayce Basques <notifications@github.com> wrote:

﻿
@kaycebasques commented on this pull request.

In src/site/content/en/blog/variable-fonts/index.md:

> +have to load a new font file for every style (or combinations of styles), many
+web developers choose not to use these capabilities, reducing the reading
+experience of their users.
+
+## Anatomy of a Variable Font
+
+Variable fonts address these challenges, by packing styles into a single file.
+
+This works by starting with a central or 'default' master style, usually the
+'Regular'–an upright roman design with the most typical weight and width that is
+most suitable for plain text. This is then connected to other style masters in a
+continuous range, called an 'axis.' The most common axis is **Weight**, which
+can connect the default style through to a Bold master style. Any individual
+style can be located along an axis, and is called an 'instance' of the variable
+font. Some instances are named by the font developer, for example Weight axis
+location 600 is called SemiBold.
@RoelN is there any other phrase we can use other than master here? My org is asking me to avoid that word because of it's political/historical connotations (master/slave)

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.

RoelN commented:
Certainly! I used @dberlow's approach. As far as I can tell, these are still smooth and sense-making sentences, but if English is your first language, please confirm :-)

dberlow commented:
If I’ve got the link correct, I’m still seeing “masters”.

![85A9CB7F-8505-43BE-9A9D-D4BAAFF323C6](https://user-images.githubusercontent.com/1956494/92103661-43742900-edae-11ea-8e45-420aa8244bae.png)

RoelN commented:
@dberlow Which link is that? When I run the site locally with the latest commit, "master" is replaced by "styles".

dberlow commented:
The link I’m following is the first line of the first message of this thread.  : https://web.dev/variable-fonts/

If that is not the link please put it there?

RoelN commented:
@dberlow The article isn't published to the live site yet.

I'm not aware of a way to see rendered versions of unpublished articles. You can see the site locally with all the updates from this PR by following [these](https://github.com/GoogleChrome/web.dev#building-the-site-) instructions and checking out this `vf-doc` branch.

dberlow commented:
How silly of me, I see. I’m sure you got the masters out just fine.

On Sep 4, 2020, at 4:49 AM, Roel Nieskens <notifications@github.com> wrote:

﻿
@dberlow The article isn't published to the live site yet.

I'm not aware of a way to see rendered versions of unpublished articles. You can see the site locally with all the updates from this PR by following these instructions and checking out this vf-doc branch.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.

RoelN commented:
@kaycebasques Anything I can help with to push this over the finish line?

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
+location 600 is called SemiBold.
+
+The variable font [Roboto Flex](https://github.com/TypeNetwork/Roboto-Flex)
+has three styles for its **Weight** axis. The Regular style is at the center,
+and two styles at the opposite ends of the axis, that are lighter and heavier.
+Between these, you can choose from 900 instances:
```
```suggestion
has three styles for its **Weight** axis. The Regular style is at the center,
and there are two styles at the opposite ends of the axis, one lighter and
the other heavier. Between these, you can choose from 900 instances:
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
-  <img src="roboto-family.png" alt="Different members of the Roboto family">
-  <figcaption class="w-figcaption">
-    Different fonts within the Roboto family.
+  <img src="roboto-flex-aaaaaa.png" style="max-width: 100%;"
```
```suggestion
  <img src="roboto-flex-aaaaaa.png"
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
-and two masters, Thinner and Bolder, are at the opposite ends of the axis.
-Between these there are potentially 200 instances that can be chosen by the
-designer or developer:
+The font developer can offer a set of different axes, and because they all share
+the same default styles, you can combine them. Roboto has three styles in a
```
```suggestion
The font developer can offer a set of different axes. You can combine them
because they all share the same default styles. Roboto has three styles in a
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
-    Typeface Amstelvar, designed by David Berlow, type designer and
-    typographer at Font Bureau.
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="roboto-dance.mp4" type="video/mp4">
```
```suggestion
    <source src="https://storage.googleapis.com/web-dev-assets/variable-fonts/roboto-dance.mp4" type="video/mp4">
```
@RoelN please delete `roboto-dance.mp4` from this PR

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
-interpolation-compatible contours, so their Roman and Italic styles can be interpolated
+This means there are thousands of styles! This may seem like massive overkill,
+but the quality of the reading experience can be remarkably enhanced by this
+diversity of type styles. And, if it is without performance penalty, web
```
My question right now as your typical web developer is: what are these performance penalties and how do I watch out for them?

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
   <img src="Amstelvar-Ital-axis.png"
   alt="Example of the Weight Axes for the typeface Amstelvar">
-  <figcaption class="w-figcaption">
+  <figcaption>
```
```suggestion
  <figcaption class="w-figcaption">
```
Need to keep our styling consistent, file a feature request with our engineering team if you want to change how captions are presented

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
+fonts and the current syntax, it will use the second declaration. They both
+point to the same font file.
+
+<!-- TODO 2021 Q1 revisit this, based on progress in https://www.w3.org/TR/css-fonts-4/#font-face-src-requirement-types to allow removing the 2nd src -->
```
```suggestion
<!-- TODO 2021 Q1 revisit this, based on progress in 
     https://www.w3.org/TR/css-fonts-4/#font-face-src-requirement-types
     to allow removing the 2nd src -->
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
+offers. The font's documentation should provide this, or you can inspect the
+font using a tool like [Wakamai Fondue](https://wakamaifondue.com).
+
+## Variable Fonts in CSS
+
+### Loading Variable Font Files
```
```suggestion
## Variable fonts in CSS

### Loading variable font files
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
+will the CSS contain the proper source formats and ranges, Google Fonts will
+also send static fallback fonts in case variable fonts aren't supported.
+
+### Using Weights and Widths
```
```suggestion
### Using weights and widths
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
+  </figcaption>
+</figure>
+
+### Using Italics and Obliques
```
```suggestion
### Using italics and obliques
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
 </figure>
 
-## Responsibility of the typesetter
+### Using Optical Sizes
```
```suggestion
### Using optical sizes
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
````diff
@@ -7,358 +7,694 @@ description: >
+}
+```
+
+### Using Custom Axes
````
```suggestion
### Using custom axes
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
````diff
@@ -7,358 +7,694 @@ description: >
+}
+```
+
+## Font-variation-settings Inheritance
````
```suggestion
## Font-variation-settings inheritance
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
-generate eight weights, across three widths, across both the Italic and Roman
-styles. Storing 48 individual fonts in a single variable font file meant a
-**88% reduction in file size**.
+## Performance Gains
```
```suggestion
## Performance gains
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
+Find more performance tips for loading Google Fonts in [Harry Roberts'
+post](https://csswizardry.com/2020/05/the-fastest-google-fonts/).
+
+## Fallbacks and Browser Support
```
```suggestion
## Fallbacks and browser support {: #fallbacks }
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
+they'll download 6 font files: one for weight 200, one for weight 300, and so
+on.
+
+## Use Cases and Benefits
```
```suggestion
## Use cases and benefits
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
+artistic designs are exciting, but for body copy this risks making the text
+illegible.
+
+### Exciting Expression
```
```suggestion
### Exciting expression
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
+that interpolate between various styles in a multi-dimensional design space.
 This gives you granular control over your typography, and a great deal of power.
 
 ## Axes Definitions
```
```suggestion
## Axes definitions
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
   <video controls autoplay loop muted class="w-screenshot">
-    <source src="https://storage.googleapis.com/web-dev-assets/variable-fonts/amstel-grade-e.mp4" 
-            type="video/mp4">
+    <source src="roboto-flex-weight.mp4" type="video/mp4">
```
```suggestion
    <source src="https://storage.googleapis.com/web-dev-assets/variable-fonts/roboto-flex-weight.mp4" type="video/mp4">
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
   </video>
-  <figcaption class="w-figcaption">
-    Grade axis being changed on the fly.
+  <figcaption>
```
```suggestion
  <figcaption class="w-figcaption">
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
-which has a series of values that pair the axis tag with an instance location:
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="roboto-flex-width.mp4" type="video/mp4">
```
```suggestion
    <source src="https://storage.googleapis.com/web-dev-assets/variable-fonts/roboto-flex-width.mp4" type="video/mp4">
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="roboto-flex-width.mp4" type="video/mp4">
+  </video>
+  <figcaption>
```
```suggestion
  <figcaption class="w-figcaption">
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
   <video controls autoplay loop muted class="w-screenshot">
-    <source src="https://storage.googleapis.com/web-dev-assets/variable-fonts/gingham-weight-e.mp4" 
-            type="video/mp4">
+    <source src="roboto-flex-slant.mp4" type="video/mp4">
```
```suggestion
    <source src="https://storage.googleapis.com/web-dev-assets/variable-fonts/roboto-flex-slant.mp4" type="video/mp4">
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
   </video>
-  <figcaption class="w-figcaption">
-    A demonstration of the Weight and Width axes being changed on the fly.
+  <figcaption>
```
```suggestion
  <figcaption class="w-figcaption">
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
-    A great example of artistic expression is 
-    <a href="https://codepen.io/mandymichael/pen/YYaWop">Mandy Michael's exploration</a> of the typeface
-    <a href="https://www.typenetwork.com/brochure/decovar-a-decorative-variable-font-by-david-berlow">Decovar</a>.
+  <img src="roboto-flex-opsz.png"
```
Please resize to `1600px` width and lightly optimize with https://squoosh.app

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
-    <a href="https://www.typenetwork.com/brochure/decovar-a-decorative-variable-font-by-david-berlow">Decovar</a>.
+  <img src="roboto-flex-opsz.png"
+  alt="The letter 'a' shown at different optical sizes">
+  <figcaption>
```
```suggestion
  <figcaption class="w-figcaption">
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
   <video controls autoplay loop muted class="w-screenshot">
-    <source src="https://storage.googleapis.com/web-dev-assets/variable-fonts/axis-praxis.mp4" 
-            type="video/mp4">
+    <source src="roboto-flex-grade.mp4" type="video/mp4">
```
```suggestion
    <source src="https://storage.googleapis.com/web-dev-assets/variable-fonts/roboto-flex-grade.mp4" type="video/mp4">
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
-    <a href="https://www.axis-praxis.org/specimens/zycon">An example of using different axes to animate characters</a>
-    with the typeface Zycon. Zycon was designed for animation by David Berlow, type designer and
-    typographer at Font Bureau.
+  <figcaption>
```
```suggestion
  <figcaption class="w-figcaption">
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
+
+CSS variables will cascade, so if an element (or one of its parents) will have
+set the `slnt` to `10`, it will keep that value, even if you set `GRAD` to
+something else. See [this
+post](https://pixelambacht.nl/2019/fixing-variable-font-inheritance/) for an
+in-depth explanation of this technique.
```
```suggestion
something else. See
[Fixing variable font inheritance](https://pixelambacht.nl/2019/fixing-variable-font-inheritance/)
for an in-depth explanation of this technique.
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
 
-{% YouTube 'B42rUMdcB7c' %}
+On the flip side, if you are animating the font between settings, this may cause
+the browser performance issues. Learn more about this on [Surma's
+Supercharged](https://www.youtube.com/watch?v=B42rUMdcB7c).
```
```suggestion
the browser performance issues. Learn more about this in
[Variable Fonts - Supercharged](https://www.youtube.com/watch?v=B42rUMdcB7c).
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
 
-## Fallbacks {: #fallbacks }
+Speaking of performance: if you're using Google Fonts, it's a good idea to
+preconnect to `https://fonts.gstatic.com`, the domain where Google's fonts are
```
```suggestion
[preconnect](/preconnect-and-dns-prefetch/) to `https://fonts.gstatic.com`,
the domain where Google's fonts are
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
````diff
@@ -7,358 +7,694 @@ description: >
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+```
+
+Find more performance tips for loading Google Fonts in [Harry Roberts'
+post](https://csswizardry.com/2020/05/the-fastest-google-fonts/).
````
```suggestion
Find more performance tips for loading Google Fonts in [The Fastest
Google Fonts](https://csswizardry.com/2020/05/the-fastest-google-fonts/).
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
 As of May 2020 variable fonts are supported in most browsers. See
 [Can I use variable fonts?](https://caniuse.com/#feat=variable-fonts)
-and [Fallbacks](#fallbacks).
+and [Fallbacks](#fallbacks-and-browser-support).
```
```suggestion
and [Fallbacks](#fallbacks).
```
We should just hardcode an explicit ID so that it stays stable over time

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
+  font-weight: bold;
+}
+
+body {
+	font-family: Roboto;
+}
+
+.super-bold {
+	font-weight: bold;
+}
```
```suggestion
body {
  font-family: Roboto;
}

.super-bold {
  font-weight: bold;
}
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
-    font-family: AmstelvarAlpha;
-    font-variation-settings: 'wdth' 400, 'wght' 98;
+  .super-bold {
+	font-weight: 1000;
```
```suggestion
    font-weight: 1000;
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
+  alt="Grass example by Mandy Michael">
+</figure>
+
+One great example of artistic expression is shown above, an explorations of the
```
```suggestion
One great example of artistic expression is shown above, an exploration of the
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="axis-praxis.mp4" type="video/mp4">
+  </video>
+  <figcaption>
```
```suggestion
    <source src="https://storage.googleapis.com/web-dev-assets/variable-fonts/axis-praxis.mp4" type="video/mp4">
  </video>
  <figcaption class="w-figcaption">
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
+the live [animation example on Axis
+Praxis](https://www.axis-praxis.org/specimens/zycon).
+
+[Anicons](https://typogram.github.io/Anicons) is the world's first animated
+[color icon font, based on Material Design Icons. Anicons is an experiment that
+[combines two cutting edge font technologies: variable fonts and color fonts.
```
```suggestion
[Anicons](https://typogram.github.io/Anicons) is the world's first animated
color icon font, based on Material Design Icons. Anicons is an experiment that
combines two cutting edge font technologies: variable fonts and color fonts.
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="anicons-animation.mp4" type="video/mp4">
+  </video>
+  <figcaption>
```
```suggestion
    <source src="https://storage.googleapis.com/web-dev-assets/variable-fonts/anicons-animation.mp4" type="video/mp4">
  </video>
  <figcaption class="w-figcaption">
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="larger-widths.mp4" type="video/mp4">
+  </video>
+  <figcaption>
```
```suggestion
    <source src="https://storage.googleapis.com/web-dev-assets/variable-fonts/larger-widths.mp4" type="video/mp4">
  </video>
  <figcaption class="w-figcaption">
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
+    <source src="larger-widths.mp4" type="video/mp4">
+  </video>
+  <figcaption>
+    Amstelvar using little bits of XTRA in opposite directions so the words widhts are evened out
```
```suggestion
    Amstelvar using little bits of XTRA in opposite directions so the words' widths are evened out
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
+these 4 aspects can be used to fine tune any other axis.
+
+The XTRA axis in Amstelvar allows you to adjust the "white" per mille value,
+as shown above. Using little bits of XTRA in opposite directions, the words
+widhts are evened out.
```
```suggestion
as shown above. Using little bits of XTRA in opposite directions, the words'
widths are evened out.
```

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,358 +7,694 @@ description: >
+as shown above. Using little bits of XTRA in opposite directions, the words
+widhts are evened out.
+
+## Thanks {: .hide-from-toc }
```
```suggestion
## Thanks
```

kaycebasques reviewed (changes requested):
- [ ] remove all MP4 files from this PR (I've uploaded them to our cloud storage bucket)

kaycebasques commented on src/site/content/en/blog/variable-fonts/index.md:
```diff
@@ -7,120 +7,131 @@ description: >
+updated: 2020-08-17
 hero: hero.jpg
 tags:
   - blog
   - fonts
 ---
```
```suggestion
  - blog
  - fonts
  - performance
  - ux
---
```

kaycebasques commented:
@RoelN almost done… please remove all MP4 files from this PR (I've uploaded them to our cloud storage bucket and replaced the links in the Markdown)

RoelN commented:
@kaycebasques Apologies for the delay! MP4 files are gone, and I resolved the latest conflicts!

kaycebasques reviewed (approved):
We're good 😎 

Thanks for your patience and for updating our guidance

RoelN commented:
Awesome! Thanks @kaycebasques 🙏
